### PR TITLE
gtk3: enabled broadway backend.

### PR DESCRIPTION
**Pull request by KrullKorg**

### CODE REVIEW

File: mingw-w64-gtk3/PKGBUILD [hunk 1]
```diff
@@ -80,4 +86,6 @@ package() {
 
   mv ${pkgdir}${MINGW_PREFIX}/bin/gtk-update-icon-cache{,-3.0}.exe
   mv ${pkgdir}${MINGW_PREFIX}/share/man/man1/gtk-update-icon-cache{,-3.0}.1
+
+  install -Dm644 "${srcdir}/gtk+-${pkgver}/COPYING" "${pkgdir}${MINGW_PREFIX}/share/licenses/gtk+3/COPYING"
```
Comment (Alexpux) [inline mingw-w64-gtk3/PKGBUILD]: Maybe just ````gtk3```` not ````gtk+3````

Comment (KrullKorg) [inline mingw-w64-gtk3/PKGBUILD]: ok; can you change it on pulling? or i have to resend the PKGBUILD?

Comment (Alexpux) [inline mingw-w64-gtk3/PKGBUILD]: You need change it yourself and do git push with ````--force````

Comment (KrullKorg) [inline mingw-w64-gtk3/PKGBUILD]: i did it

**Pull request merged**